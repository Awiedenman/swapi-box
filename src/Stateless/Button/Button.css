.button-div {
  display: inline-block;
  }

.button {
  width: 100px;
  height: 30px;
  margin: 0 22px;
  background-color: rgb(25, 26, 26);
  color:rgb(180, 187, 84);
  border: 3px rgb(180, 187, 84) inset;
}

.button:hover{
background-color: rgb(133, 58, 108);
 opacity: .7;

}