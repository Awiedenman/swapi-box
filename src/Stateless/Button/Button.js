import React from 'react';
import './Button.css'

const Button = ( props ) => {
  return (
    <div>
      <button></button>
    </div>
  )
}