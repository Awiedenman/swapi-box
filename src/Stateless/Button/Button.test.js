
import React from 'react';
import Button from './Button';
import { shallow, mount } from 'enzyme';

describe('Button', () => {
  let wrapper;
  // const mockSetData = jest.fn()
 
  beforeEach( () => {
    wrapper = shallow( 
          <Button
            name= {name}
            // setData={ setData}
          />
           )
  })

  it('should match the snapshot of the button on page',() =>{
    expect(Button).toMatchSnapshot()
  })
})