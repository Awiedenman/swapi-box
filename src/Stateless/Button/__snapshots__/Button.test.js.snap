// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match the snapshot of the button on page 1`] = `[Function]`;
